app/src/main/java/a3/com/convo/activities/PlayGameActivity.java

```diff
@@ -6,38 +6,38 @@
 import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
 
+import java.util.ArrayList;
+
 import a3.com.convo.R;
+import a3.com.convo.fragments.ConclusionFragment;
 import a3.com.convo.fragments.FriendsFragment;
 import a3.com.convo.fragments.GameFragment;
 import a3.com.convo.fragments.ModeFragment;
 
 public class PlayGameActivity extends AppCompatActivity {
 
-    private FriendsFragment friendsFrag;
-    private ModeFragment modeFrag;
-    private GameFragment gameFrag;
-    private Fragment playGameFrag;
     private FragmentTransaction ft;
```

Comment:
We dont need this as member variable, we can call getSupportFragmentManager() at any point to get the required transaction.